.github/workflows/falcor-compiler-perf-test.yml

```diff
@@ -0,0 +1,52 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Falcor Compiler Perf-Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  build:
+    runs-on: [Windows, self-hosted, perf]
+    timeout-minutes: 100
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        configuration: ['Release']
+        platform: ['x64']
+        include:
+          - platform: x64
+            testPlatform: x64
+          - platform: x64
+            testCategory: full
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: 'true'
+        fetch-depth: '0'
+    - name: setup-msbuild
+      uses: microsoft/setup-msbuild@v1
+    - name: build
+      run: |
+        .\premake.bat vs2019 --arch=${{matrix.platform}} --deps=true --no-progress=true --enable-cuda=true
+
+        .\make-slang-tag-version.bat
+
+        MSBuild.exe slang.sln -v:m -m -property:Configuration=${{matrix.configuration}} -property:Platform=${{matrix.platform}} -property:WindowsTargetPlatformVersion=10.0.19041.0 -maxcpucount:12
+
+    - name: download falcor-compiler-perf-test
```

Comment:
Use a release downloader action to automatically grab the latest release:
https://github.com/marketplace/actions/release-downloader

Reply:
Done.